flottform/demo/src/routes/expense-report/+page.svelte

```diff
@@ -0,0 +1,614 @@
+<script lang="ts">
+	import { base } from '$app/paths';
+	import { onMount } from 'svelte';
+	import { FlottformFileInputHost } from '@flottform/forms';
+	import { writable } from 'svelte/store';
+	import { env } from '$env/dynamic/public';
+	import { browser } from '$app/environment';
+	import { sdpExchangeServerBase } from '../../api';
+	import waiting from './images/status-svgs/waiting.svg';
+	import done from './images/status-svgs/done.svg';
+	import errorSvg from './images/status-svgs/error.svg';
+
+	let fileInput: HTMLInputElement;
+	let flottformButton: HTMLButtonElement;
+	let flottformDialogCardButton: HTMLButtonElement;
+	let flottformDialogCard: HTMLDialogElement;
+	let copyToClipboardButton: HTMLButtonElement;
+	let flottformStatusWrapper: string = $state('');
+	let flottformQrCode: string = $state('');
+	let flottformLinkOffer: string = $state('');
+	let flottformDialogDescription: string = $state('');
+
+	let isFlottformDialogOpened: boolean = $state(false);
+
+	let flottformState: string = $state('');
+	let flottformStatusSvg: string = $state('');
+
+	let isEndpointCreated = false;
+
+	let createWebRtcChannel: () => void;
+
+	const openFlottformDialogCard = () => {
+		flottformDialogCard.showModal();
+		isFlottformDialogOpened = true;
+	};
+
+	const closeFlottformDialogCard = () => {
+		flottformDialogCard.close();
+		isFlottformDialogOpened = false;
+	};
+
+	const handleFlottformButtonClick = () => {
+		if (!isEndpointCreated) {
+			createWebRtcChannel();
+		}
+	};
+
+	const clientBase = env.PUBLIC_FLOTTFORM_CLIENT_BASE || 'https://192.168.0.21:5177/custom-ui';
```

Comment:
`custom-ui`? 🤔

I think we should remove the IP addresses in these parts as we've used them during the prototype / Hackathon phase. Would be better to have the env variables documented